src/kit.ts

```diff
@@ -1108,29 +1108,38 @@ export async function descriptionForKit(kit: Kit, shortVsName: boolean = false):
     return localize('unspecified.let.cmake.guess', 'Unspecified (Let CMake guess what compilers and environment to use)');
 }
 
+async function expandKitVariable(variable: string, kitName: string): Promise<string> {
+    return expand.expandString(variable, {
```

Comment:
Splitting this out into a separate function means that the object passed in as the second argument has to be created every time it is called.  I think it would be better to create a temporary variable in `expandKitVariables` and just call `expand.expandstring` directly in there.